map_view/show-location-history/README.metadata.json

```diff
@@ -3,7 +3,7 @@
     "description": "Display your location history on the map.",
     "ignore": false,
     "images": [
-        "ShowLocationHistory.png"
+        "ShowLocationHistory.gif"
     ],
     "keywords": [
         "GPS",
```

Comment:
we should remove the following classes from keywords and relevant API section here since we have removed them from the readme: 

* LocationDataSource	
* LocationDataSource.Location
* LocationDataSource.Position